test/testcase/primitive/String.test.ts

```diff
@@ -3,18 +3,18 @@ import {
   guessCWD,
   loadConfig,
   processConfig,
-  TestCaseSampler,
   setupLogger,
   setupOptions,
-} from "@syntest/framework";
+} from "@syntest/core";
 import { StringStatement } from "../../../dist/testcase/statements/primitive/StringStatement";
+import { SoliditySampler } from "../../../dist/testcase/sampling/SoliditySampler";
```

Comment:
Still has some references to dist

Reply:
Fixed although this is the dist of the current package